### ISSUE TITLE
Login failed error when using options.tokenStorage = session

### ISSUE THREAD

**Issue by warmwaterkruik** [UniversalViewer/universalviewer#448]
We currently have an open universalviewer implementation without logins. But 
we get the you should login error message inside the viewer or something
like "Your log-in attempt did not appear to be successful. Please try again"

Temporary fix:
Set the json property options.tokenStorage = 'memory' inside the file
uv-seadragon-extension.nl.config.json. Reload the page with browser caches
cleared and the error should go away.
The problem is not an authentication error, but the viewer trying to read
the window.sessionStorage object and finding Drupal storage keys with html
data but treating them like JSON. JSON throws a could not parse html exception
but the viewer incorrectly catches that as an authentication failed error.
In a nutshell; the viewer does not handle session keys properly and fails.

Longterm fix:
The problem is in the external dependency utils/Storage class that tries to parse values in window.sessionStorage as JSON data, but sessionStorage can also used by the site that stores scalars or html values. Considering that is a dependency we could change the BaseExtension::getStoredAccessToken() method to change the token loading strategy or wait for the change request at edsilv/utils : https://github.com/edsilv/utils/issues/5
